schema/artwork.js

```diff
@@ -11,6 +15,21 @@ import {
   GraphQLInt
 } from 'graphql';
 
+export let ArtworkPredicates = {
+  is_contactable: (artwork, relatedSales) => {
+    return artwork.forsale && !_.isEmpty(artwork.partner) && !artwork.acquireable && !relatedSales.length
+  }
+}
+
+let fetchRelatedSales = ({ id }, options) => {
+  options = qs.stringify(_.defaults(options, {
+    size: 1,
+    active: true,
```

Comment:
But there's a mystery here!

If these results potentially included inactive sales all this time, have clients already worked around this, or is it a real user-facing bug?